openfisca_web_api/controllers/entities.py

```diff
@@ -9,19 +9,8 @@
 from .. import contexts, conv, model, wsgihelpers
 
 
-def build_entity_data(entity):
-    entity_data = {'label': entity.label}
-    if entity.is_person:
-        entity_data['isPersonsEntity'] = entity.is_person
-    else:
-        entity_data.update({
-            'roles': entity.roles_description,
-            })
-    return entity_data
-
-
 @wsgihelpers.wsgify
-def api1_entities(req):
+def api2_entities(req):
```

Comment:
Hmm, this is to follow the naming in use (`/api/N/name` => `apiN_name`): see [`controllers/__init__.py`](https://github.com/openfisca/openfisca-web-api/blob/c7a16ff786c1869a9e7bc7aed142c8f4b6cd0b7a/openfisca_web_api/controllers/__init__.py#L37-L48).